Makefile

```diff
@@ -235,9 +235,18 @@ deploy-test: _verify_minikube generate-keys
 
 deploy-onprem:
 	podman pod create --name assisted-installer -p 5432,8000,8090,8080
+	# These are required because when running on RHCOS livecd, the coreos-installer binary and
+	# livecd are bind-mounted from the host into the assisted-service container at runtime.
+	[ -f livecd.iso ] || curl $(BASE_OS_IMAGE) -o livecd.iso
```

Comment:
What do you think about using livecd.iso from quay.io/ocpmetal/assisted-iso-create? The container should have been built locally in the CI so we would just need to extract the iso from that container.

Reply:
I didn't go that route because my hypothesis is that the assisted-iso-create container is not required (https://issues.redhat.com/browse/MGMT-2636).